frontend/src/scenes/surveys/SurveyView.scss

```diff
@@ -17,3 +17,9 @@
     margin-bottom: 10px;
     break-inside: avoid;
 }
+
+.survey-view {
+    .survey-form {
+        z-index: 0 !important;
```

Comment:
can we use one of the existing z-index variables? Right now this will do poorly with other app interactions, like for example a modal showing up for whatever reason

https://github.com/PostHog/posthog/blob/master/frontend/src/styles/vars.scss#L248

Reply:
but which z-index would you pick from the list? imo this survey preview shouldn't be raised at all, i.e. z-index should be 0 🤔 

a modal would showing over the survey preview is okay I think

Reply:
🤦 d'oh, this is on the preview, not on the dropdown button, silly me! yeah much nicer fix this is 😂